spot-client/package.json

```diff
@@ -73,6 +72,7 @@
         "react-router-dom": "5.1.2",
         "redux": "4.0.5",
         "redux-thunk": "2.3.0",
+        "sass": "1.54.9",
```

Comment:
Please split this change to a separate commit. One logical change one commit / PR.